solutions/ova/kubernetes/ova-solution.yaml

```diff
@@ -11,29 +11,26 @@ solution:
       csm_auth_admin_secret: seagate2
```

Comment:
Please remove all the secrets except below
```
      s3_auth_admin_secret: ldapadmin
      csm_mgmt_admin_secret: Cortxadmin@123
```

Reply:
@shailesh-vaidya, solution.yaml is already validated for deployment i.e. https://eos-jenkins.colo.seagate.com/job/Cortx-Deployment/job/Nightly-K8s-Deploy/790/console

Reply:
@shailesh-vaidya, This has been validated with jenkins job, https://eos-jenkins.colo.seagate.com/job/Cortx-Kubernetes/job/setup-cortx-cluster-solution-input/449/